prow/prowjobs/GoogleCloudPlatform/gcp-guest/gcp-guest-config.yaml

```diff
@@ -19,10 +19,29 @@ postsubmits:
           secretName: compute-image-tools-test-service-account
     max_concurrency: 10
     branches:
-      - ^master$
-
+    - ^master$
 presubmits:
   GoogleCloudPlatform/osconfig:
+  - name: osconfig-presubmit-build
+    cluster: gcp-guest
+    trigger: "(?m)^/osconfigbuild$"
+    rerun_command: "/osconfigbuild"
+    context: prow/presubmit/gotest
+    decorate: true
+    spec:
+      containers:
+      - image: gcr.io/gcp-guest/daisy_build_upload:latest
+        command:
+        - "/daisy_build_upload.sh"
+        args: ["gcp-guest", "us-west1-a", "./packaging/build_packages_wf.json", "gs://osconfig-agent-package/"]
+        volumeMounts:
+        - name: compute-image-tools-test-service-account
```

Comment:
rename all references from c-i-t-test-service-account to gcp-guest-svc-acct

Reply:
daisy-service-account

Reply:
changed